carousel.js

```diff
@@ -0,0 +1,62 @@
+
+
+class Carousel {
+	constructor(element) {
+		this.element = element;
+		this.arrows = element.querySelectorAll(".Arrows");
+		this.counter = 0;
+		this.pages = new Pages(this.element);
+		Array.from(this.arrows).forEach((arrow) => {
+			arrow.addEventListener('click', () => {
+				let previousCounter = this.counter;
+				if (arrow.classList.contains("Arrow--left")) {
+					console.log("left arrow clicked");
+					this.counter--;
+				} else if (arrow.classList.contains("Arrow--right")) {
+					console.log("right arrow clicked");
+					this.counter++;
+				}
+				if (this.counter === -1) {
+					this.counter = 3;
+				}
+				if (this.counter === 4) {
+					this.counter = 0;
+				}					
+			//	console.log(this.counter);
+				this.pages.select(this.counter);
+				this.pages.deselect(previousCounter);
+			});
+		});
+	}
+}
+
+class Pages {
+	constructor(element) {
+		this.element = element;
+		this.pages = element.querySelectorAll(".Pages__images");
```

Comment:
This won't be needed if you follow my previous comment. The pages element would be assigned to "this.element" on line 35.